syncopy/connectivity/AV_compRoutines.py

```diff
@@ -289,14 +289,14 @@ def pre_check(self):
             lgl = 'Initialize the computational Routine first!'
             act = 'ComputationalRoutine not initialized!'
             raise SPYValueError(legal=lgl, varname=self.__class__.__name__, actual=act)
-        
+
         if self.numTrials != 1:
             lgl = "1 trial: normalizations can only be done on averaged quantities!"
             act = f"DataSet contains {self.numTrials} trials"
             raise SPYValueError(legal=lgl, varname="data", actual=act)
-    
+
     def process_metadata(self, data, out):
-        
+
         # Get trialdef array + channels from source
         if data._selection is not None:
             chanSec = data._selection.channel
```

Comment:
The datatype here still is `CrossSpectralData` no? How/why does a `._selection.channel` work then here?

Reply:
Good catch. Oversight on my end - I assumed, that `data` is an `AnalogData` object (as in the `ST_` routines). For `CrossSpectralData` the above channel selection does not work. I'll patch this.

Reply:
Fix has been pushed (cf 61f2ee3)